Neos.Fusion/Classes/FusionObjects/UriBuilderImplementation.php

```diff
@@ -79,7 +79,8 @@ public function getAction()
      */
     public function getArguments()
```

Comment:
```suggestion
    public function getArguments(): array
```

Reply:
I left the return type declaration out, since we don't have that in the class in this branch. Consistency wins in that case.